orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -200,13 +201,35 @@ def __init__(self, module: _FlattenedModule, debug_options: DebugOptions, fallba
 
         self._print_input_density = ortmodule._defined_from_envvar("ORTMODULE_PRINT_INPUT_DENSITY", 0, warn=True) == 1
 
+        self._enable_memory_optimizer = ortmodule._defined_from_envvar("ORTMODULE_MEMORY_OPT_CONFIG", "", warn=True)
+
         # Flag to re-export the model due to attribute change on the original module.
         # Re-export will be avoided if _skip_check is enabled.
         self._original_model_has_changed = False
 
         # Load ATen operator executor extension.
         load_aten_op_executor_cpp_extension()
 
+        self._feature_map: List[List[str]] = [
+            ["ATen Executor", "ON", "Dispatch ATen operators to ORT's ATen executor"],
+            [
+                "Cast Propagation",
+                "ON" if self._propagate_cast_ops_level > 0 else "OFF",
+                f"Level {self._propagate_cast_ops_level} enabled",
+            ],
+            ["Custom Function", "ON", "Support custom torch.autograd.Function export and execution"],
+            [
+                "Compute Optimizer",
```

Comment:
Does this cover label sparsity optimizations? If not shall we add it here?

Reply:
Oh, currently sparsity optimizations is listed as a seperated row. 

I have move sparse opt as a sub-feature of compute optimizer to represent the control scope of those features.